This change is related to:

http://git.dcmtk.org/web?p=dcmtk.git;a=commitdiff;h=1d751592ff858b1dcb0f82647c06de779b8e7c32

It allows compilation against DCMTK 3.6.0
Index: aeskulap-0.2.2b1/imagepool/dicomdir.cpp
===================================================================
--- aeskulap-0.2.2b1.orig/imagepool/dicomdir.cpp	2011-02-14 10:27:07.000000000 +0100
+++ aeskulap-0.2.2b1/imagepool/dicomdir.cpp	2011-02-14 10:27:10.000000000 +0100
@@ -151,21 +151,21 @@
             DcmDataset study;
             DcmElement *el;
 
-            if ( studyRec->findAndCopyElement(DCM_SpecificCharacterSet, el) == ECC_Normal )
+            if ( studyRec->findAndGetElement(DCM_SpecificCharacterSet, el, OFFalse, OFTrue) == ECC_Normal )
                 study.insert(el);
-            if ( studyRec->findAndCopyElement(DCM_StudyInstanceUID, el) == ECC_Normal )
+            if ( studyRec->findAndGetElement(DCM_StudyInstanceUID, el, OFFalse, OFTrue) == ECC_Normal )
                 study.insert(el);
-            if ( studyRec->findAndCopyElement(DCM_StudyDate, el) == ECC_Normal )
+            if ( studyRec->findAndGetElement(DCM_StudyDate, el, OFFalse, OFTrue) == ECC_Normal )
                 study.insert(el);
-            if ( studyRec->findAndCopyElement(DCM_StudyTime, el) == ECC_Normal )
+            if ( studyRec->findAndGetElement(DCM_StudyTime, el, OFFalse, OFTrue) == ECC_Normal )
                 study.insert(el);
-            if ( studyRec->findAndCopyElement(DCM_StudyDescription, el) == ECC_Normal )
+            if ( studyRec->findAndGetElement(DCM_StudyDescription, el, OFFalse, OFTrue) == ECC_Normal )
                 study.insert(el);
-            if ( patRec->findAndCopyElement(DCM_PatientsName, el) == ECC_Normal )
+            if ( patRec->findAndGetElement(DCM_PatientName, el, OFFalse, OFTrue) == ECC_Normal )
                 study.insert(el);
-            if ( patRec->findAndCopyElement(DCM_PatientsBirthDate, el) == ECC_Normal )
+            if ( patRec->findAndGetElement(DCM_PatientBirthDate, el, OFFalse, OFTrue) == ECC_Normal )
                 study.insert(el);
-            if ( patRec->findAndCopyElement(DCM_PatientsSex, el) == ECC_Normal )
+            if ( patRec->findAndGetElement(DCM_PatientSex, el, OFFalse, OFTrue) == ECC_Normal )
                 study.insert(el);
             resultslot(create_query_study(&study, std::string("DICOMDIR:") + dicomdir));
         }
@@ -221,21 +221,21 @@
             DcmDataset series;
             DcmElement *el;
 
-            if ( seriesRec->findAndCopyElement(DCM_SpecificCharacterSet, el) == ECC_Normal )
+            if ( seriesRec->findAndGetElement(DCM_SpecificCharacterSet, el, OFFalse, OFTrue) == ECC_Normal )
                 series.insert(el);
-            if ( seriesRec->findAndCopyElement(DCM_SeriesDescription, el) == ECC_Normal )
+            if ( seriesRec->findAndGetElement(DCM_SeriesDescription, el, OFFalse, OFTrue) == ECC_Normal )
                 series.insert(el);
-            if ( seriesRec->findAndCopyElement(DCM_SeriesInstanceUID, el) == ECC_Normal )
+            if ( seriesRec->findAndGetElement(DCM_SeriesInstanceUID, el, OFFalse, OFTrue) == ECC_Normal )
                 series.insert(el);
-            if ( seriesRec->findAndCopyElement(DCM_Modality, el) == ECC_Normal )
+            if ( seriesRec->findAndGetElement(DCM_Modality, el, OFFalse, OFTrue) == ECC_Normal )
                 series.insert(el);
-            if ( seriesRec->findAndCopyElement(DCM_SeriesDate, el) == ECC_Normal )
+            if ( seriesRec->findAndGetElement(DCM_SeriesDate, el, OFFalse, OFTrue) == ECC_Normal )
                 series.insert(el);
-            if ( seriesRec->findAndCopyElement(DCM_SeriesTime, el) == ECC_Normal )
+            if ( seriesRec->findAndGetElement(DCM_SeriesTime, el, OFFalse, OFTrue) == ECC_Normal )
                 series.insert(el);
-            if ( studyRec->findAndCopyElement(DCM_StudyDescription, el) == ECC_Normal )
+            if ( studyRec->findAndGetElement(DCM_StudyDescription, el, OFFalse, OFTrue) == ECC_Normal )
                 series.insert(el);
-            if ( studyRec->findAndCopyElement(DCM_StationName, el) == ECC_Normal )
+            if ( studyRec->findAndGetElement(DCM_StationName, el, OFFalse, OFTrue) == ECC_Normal )
                 series.insert(el);
 
             // Count Related SOP Instances
